src/attributes_doc/_base.py

```diff
@@ -54,6 +60,29 @@ def get_attributes_doc(cls):
 
 def attributes_doc(cls):
     # type: (Type[T]) -> Type[T]
+    """Store the docstings of the attributes of a class in attributes named `__doc_NAME__`."""
     for attr_name, attr_doc in get_attributes_doc(cls).items():
-        setattr(cls, '__doc_%s__' % attr_name, attr_doc)
+        setattr(cls, "__doc_%s__" % attr_name, attr_doc)
     return cls
+
+
+def enum_doc(cls):
+    # type: (Type[T]) -> Type[T]
+    """Store the docstrings of the vaules of an enum in their `__doc__` attribute."""
+    for attr_name, attr_doc in get_attributes_doc(cls).items():
+        getattr(cls, attr_name).__doc__ = attr_doc
+    return cls
+
+
+def getdoc(obj, attr_name):
```

Comment:
Maybe rename this function to `get_doc`? Other functions have names in this style.

Reply:
Sure, I was going with imitating `getattr` but I guess it makes no difference